mlir/include/mlir/Dialect/MQTOpt/Transforms/Passes.td

```diff
@@ -46,6 +46,14 @@ def MergeRotationGates : Pass<"merge-rotation-gates", "mlir::ModuleOp"> {
   }];
 }
 
+def ElidePermutations : Pass<"elide-permutations", "mlir::ModuleOp"> {
+  let summary = "This pass searches for consecutive applications of rotation gates that can be merged.";
+  let description = [{
+    Remove SWAP gates and permutate order of Qubits instead.
+    This optimization does only work for unmapped circuits which do not have any connectivity restraints.
```

Comment:
A little more detailed explanation would not hurt here. This eventually ends up in our documentation (which will soon also be published as part of #1147).
What is here in the description could be the brief at the moment.

I'd turn the brief into something like
> Elides permutations (i.e., SWAP gates) by removing them from the circuit and, instead, permuting the respective qubit values.

Then, in the description, it would be great to have a fairly brief example of the pass in action. (MWE-style)
I'd also slightly rephrase the note about connectivity to rather have it point out that this optimization is not connectivity-preserving.